LiskHQ/lisk-desktop
# Bug fix: Electron app does not validate URLs on new windows and navigation

### What was the problem?

This PR resolves #5284

### How was it solved?

- [x] Added protocol checks on navigating to an external link

### How was it tested?

- Start up the electron app using this command `yarn build:electron && yarn start`
- Open up the dev console `command + shift + i`
- Execute this snippet on the console `location.href = 'https://lisk.com'`
- It would be observed that this link would be opened in default browser 
- Try again by executing this snippet on the console `location.href = 'http://lisk.com'`
- it should be observed that this link would not be navigated to.
